portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -62,12 +62,36 @@
 
     <!-- filters -->
     <filter>
-      <filter-name>CorsFilter</filter-name>
-      <filter-class>org.apache.catalina.filters.CorsFilter</filter-class>
-      <init-param>
-        <param-name>cors.exposed.headers</param-name>
-        <param-value>total-count</param-value>
-      </init-param>
+        <filter-name>CorsFilter</filter-name>
+        <filter-class>org.apache.catalina.filters.CorsFilter</filter-class>
+        <init-param>
+            <param-name>cors.allowed.origins</param-name>
+            <param-value>*</param-value>
+        </init-param>
+        <init-param>
+            <param-name>cors.allowed.methods</param-name>
+            <param-value>GET,POST,HEAD,OPTIONS</param-value>
+        </init-param>
+        <init-param>
+            <param-name>cors.allowed.headers</param-name>
+            <param-value>Origin,Accept,X-Requested-With,Content-Type,Access-Control-Request-Method,Access-Control-Request-Headers</param-value>
+        </init-param>
+        <init-param>
+            <param-name>cors.exposed.headers</param-name>
+            <param-value>total-count</param-value>
+        </init-param>
+        <init-param>
+            <param-name>cors.support.credentials</param-name>
+            <param-value>false</param-value>
```

Comment:
@n1zea144, @mandawilson  Do I understand it correctly that this makes the backend less vulnerable to CSRF?

Reply:
Hi @pieterlukasse Sorry I missed your comment here.  Based on author comments, this PR is just reenabling default settings that were lost when cors.exposed.headers was added to web.xml (I wasn't aware of this behavior - the filter dropping all default behavior when a setting is overridden, but looking at the tomcat docs, these are default settings).  

No, this setting does not make the backend less vulnerable.  If we did have CSRF protection turned on in the backend, CORS could prevent a token from being read by a different domain,  but CSRF works by tricking the user into making the forged request (same origin).